pkg/server/plugin/nodeattestor/gcp/iit.go

```diff
@@ -58,11 +59,12 @@ type IITAttestorPlugin struct {
 
 // IITAttestorConfig is the config for IITAttestorPlugin.
 type IITAttestorConfig struct {
-	idPathTemplate      *template.Template
-	trustDomain         string
-	ProjectIDWhitelist  []string `hcl:"projectid_whitelist"`
-	AgentPathTemplate   string   `hcl:"agent_path_template"`
-	UseInstanceMetadata bool     `hcl:"use_instance_metadata"`
+	idPathTemplate          *template.Template
+	trustDomain             string
+	ProjectIDWhitelist      []string `hcl:"projectid_whitelist"`
+	AgentPathTemplate       string   `hcl:"agent_path_template"`
+	UseInstanceMetadata     bool     `hcl:"use_instance_metadata"`
+	ServiceAccountTokenPath string   `hcl:"service_account_file"`
```

Comment:
There is some inconsistency in the naming, both between the variable name and HCL name and the variable as it's passed to different functions throughout the plugin. We should be consistent. I'd suggest standardizing on `service_account_file`.